src/audio/igo_nr/igo_nr.c

```diff
@@ -402,7 +402,7 @@ static int32_t igo_nr_params(struct comp_dev *dev,
 	cd->source_rate = audio_stream_get_rate(&source_c->stream);
 	cd->sink_rate = audio_stream_get_rate(&sink_c->stream);
 
-	if (audio_stream_get_channels(source_c->stream) !=
+	if (audio_stream_get_channels(&source_c->stream) !=
```

Comment:
interesting, this part of code should not be compiled with current build system, compiler will report error with this cse.

Reply:
Yes, it's because it's not test in CI currently, trying to fix that :)